server/controller/routes/postAnswers.js

```diff
@@ -0,0 +1,23 @@
+const addAnswer = require("../../db/queries/postAnswers");
+
+module.exports = (req, res, next) => {
+  try {
+
+    // this is if we send all the answers as a object in the post body
+    // the structure needs to be { answer1, answer2, answer3 } = req.body.data;
+
+    const dataToSend = req.body.data;
```

Comment:
The name dataTo Send is not clear 
```suggestion
    const answers = req.body.data;
```